README.md

```diff
@@ -44,6 +45,38 @@ release.
 
 An example of such `release.yml` can be found at: https://github.com/snowdrop/spring-boot-bom/blob/sb-2.3.x/release.yml
 
+### Init release repositories for a new Major.Minor
+
+Whenever a new Major.Minor release occur some of the git repositories used in the release process need some restructuring.  
+For instance, a new `sb-<major>.<minor>.x` branch must be created in the `snowdrop/spring-boot-bom` github repostory and a new
+folder in the `middleware/build-configurations` gitlab repository must also be created.
+
+This command performs these operations automatically.  
+
+```bash
+java -jar target/issues-manager-$(xpath -q -e  "/project/version/text()" pom.xml)-runner.jar \
+    -u <jira user> -p <jira password> 
+    new-maj-min 
+    -g <github org>/<github repo>/<git reference: branch, tag, hash> \ 
+    -o <github token> \
+    -glu <gitlab user> \
+    -glt <gitlab token> \ 
+    -r <new release> \
+    -pr <previous release>
```

Comment:
- Can you add a remark to explain why we must pass as parameter the previous release ?
- What should be the format/syntax of the parameter: `release` and `previous release` ?
- What will happen if no `previous release` exists ?

Reply:
Implemented - https://github.com/jacobdotcosta/release-manager/tree/init-release-folders#init-release-repositories-for-a-new-majorminor